pym/bob/scm/git.py

```diff
@@ -250,9 +250,8 @@ async def __checkoutTag(self, invoker, fetchCmd, switch):
                     ok = await invoker.callCommand(fetchCmd + [self.__commit],
                         cwd=self.__dir)
                     if ok != 0:
-                        self.fail("Plain git-fetch in", self.__dir,
-                            "did not download the requested commit and the explicit fetch failed!",
-                            returncode=ok)
+                        raise BuildError("Plain git-fetch in " + self.__dir +
+                            " did not download the requested commit and the explicit fetch failed!")
```

Comment:
The fail method is on the invoker:

```suggestion
                        invoker.fail("Plain git-fetch in", self.__dir,
                            "did not download the requested commit and the explicit fetch failed!",
                            returncode=ok)
```